### ISSUE TITLE
Tabs aren't keyboard accessible.

### ISSUE THREAD

**Issue by pavkout** [getinsomnia/insomnia#4942]
### Expected Behavior

Hello Insomnia,

When I open a document I should be able to navigate between different tabs via keyboard (Tab, Shift + Tab, Left Arrow, Right Arrow, Up Arrow and Down Arrow).

For instance here

![Screenshot 2022-07-07 at 9 45 38 PM](https://user-images.githubusercontent.com/1693892/177849486-e1052f6a-4777-4e40-b79a-8ea747e0179f.png)

or here 

![Screenshot 2022-07-07 at 9 47 26 PM](https://user-images.githubusercontent.com/1693892/177850168-94a97783-5cdc-43ab-a05e-dd19cf93dcdd.png)

### Actual Behavior

Currently, this is not supported.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2022.5.0-beta.2

### What operating system are you using?

macOS

### Operating System Version

macOS Monterey

### Installation method

download from insomnia.rest

**Comment by filfreire** [comment]
@pavkout check [my comment here](https://github.com/Kong/insomnia/pull/4944#issuecomment-1192455911).

**Comment by pavkout** [comment]
Hey @filfreire, please check my [comment](https://github.com/Kong/insomnia/pull/4944#discussion_r927560919).
